testing/dart/window_hooks_integration_test.dart

```diff
@@ -426,4 +426,16 @@ void main() {
     expectEquals(window.padding.bottom, 0.0);
     expectEquals(window.systemGestureInsets.bottom, 44.0);
   });
+
+  test('PlatformDispatcher.locale returns unknown locale when locales is set to empty list', () {
+    Locale? locale;
+    runZoned(() {
+      window.onLocaleChanged = () {
+        locale = PlatformDispatcher.instance.locale;
+      };
+    });
+
+    _updateLocales(<String>[]);
+    expectEquals(locale, const Locale.fromSubtags());
```

Comment:
The API docs say that we return a Locale with language tag 'und', but https://api.flutter.dev/flutter/package-intl_locale/Locale/fromSubtags.html doesn't seem to guarantee that. Maybe we should be checking as much here?

Reply:
Or we could change the docs for `fromSubtags`...

Reply:
I did both.